tests/foreman/api/test_capsulecontent.py

```diff
@@ -1651,6 +1665,112 @@ def test_positive_content_counts_blank_update(
             counts is None or len(counts['content_view_versions']) == 0
         ), f"No content counts expected, but got:\n{counts['content_view_versions']}."
 
+    @pytest.mark.skip_if_open('SAT-25542')
```

Comment:
should this be turned to `:BlockedBy` or `:Verifies` tag?

Reply:
`:BlockedBy:`

Reply:
`:Verifies:` is primarily used to map a feature to test(s). `:BlockedBy:` for skipping a test because of some bugs.

Reply:
Added `:BlockedBy:`.